index.html

```diff
@@ -8,11 +8,143 @@
     <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
   </head>
   <body>
-    <nav class="navbar">Watcher</nav>
-    <div class="video">Video</div>
+    <nav class="navbar">
```

Comment:
Please redo the navbar so it looks more youtube-like. Use fontawsome for icons if you like, and add the search field.